VirtualH89/Src/AddressBus.cpp

```diff
@@ -41,82 +33,36 @@ AddressBus::readByte(WORD addr,
         return ic_m->readDataBus();
     }
 
-    BYTE index = addr >> pageSizeBits_c;
-
-    debugss(ssAddressBus, ALL, "%s: addr(%d), index(%d)\n", __FUNCTION__, addr, index);
+    // Could do this entirely inside MemoryLayout, but this way we have
```

Comment:
I don't see a use case to have address bus be able to access banks other than the current bank. I could see that being useful for some complete save/restore of the system. It seems like we could have another readByte(addr), which just grabs from the current bank. We can keep the readByte(bnk, addr) for other classes.